index.html

```diff
@@ -12,5 +12,14 @@
 
   <body>
     <div id="content"></div>
+    <div id="chat">
+        <ul id="chatBox">
+        </ul>
+        <input id="chatInput" type="text" placeholder="Chat here..." maxlength="35" />
+    </div>
+    <div id="login">
+      Name: <input type="text" tabindex="0" autofocus placeholder="Enter your name here" id="userNameInput" maxlength="25" />
```

Comment:
Put the name text iinto a label tag:
`<label for="userNameInput">Name:</label>`